doc/nrf/known_issues.rst

```diff
@@ -2191,6 +2191,16 @@ Trusted Firmware-M (TF-M)
 
 .. rst-class:: v2-0-0
 
+NCSDK-15909: TF-M may fail to build because of flash overflow with Zephyr SDK 0.14.2
+  when TFM_PROFILE_TYPE_NOT_SET=y with TF-M.
```

Comment:
```suggestion
NCSDK-15909: TF-M may fail to build due to flash overflow with Zephyr SDK 0.14.2
  when TFM_PROFILE_TYPE_NOT_SET=y with TF-M.
```

Also, for some reason "when TFM_PROFILE_TYPE_NOT_SET=y with TF-M" sounds a bit strange to me. Would it work to replace "with TF-M" with "for TF-M"?

Reply:
I applied your recommendation and removed the "with TF-M". I think that it looks better like this but let me know if you disagree.